net-wireless/nitroshare/nitroshare-0.4.0~git.recipe

```diff
@@ -60,15 +67,19 @@ defineDebugInfoPackage nitroshare$secondaryArchSuffix \
 
 BUILD()
 {
-	cmake -DCMAKE_INSTALL_PREFIX:PATH=$appsDir .
+	cmake -DCMAKE_BUILD_TYPE=Release \
+	-DINSTALL_INCLUDE_PATH=$includeDir
+	
 	make $jobArgs
 }
 
 INSTALL()
 {
+	make install
+	rm -rf $includeDir
 	mkdir -p $appsDir
-	cp out/nitroshare $appsDir/Nitroshare
+	cp out/bin/nitroshare-ui $appsDir/Nitroshare
```

Comment:
Maybe create a symbolic link instead of copy?

Reply:
All good.